slides/r-adoption-jan2023.qmd

```diff
@@ -0,0 +1,229 @@
+---
+title: "MMRM and ASA SWE - R Adoption Series"
```

Comment:
Can we change this to the title on the website:
```suggestion
title: "Introducting the Software Engineering Working Group and {mmrm}"
```
the R Adoption series can be subtitle e.g. or date